content/en/methods/instance/trends.md

```diff
@@ -6,6 +6,9 @@ menu:
     weight: 10
     parent: methods-instance
 ---
+**Approved hashtags**
+- show trending hashtags on the side of the webui
```

Comment:
It seems completely irrelevant how the web UI uses this API on the API documentation.